src/_locales/ru/messages.json

```diff
@@ -60,28 +60,28 @@
     "html_options_suspend_never": { "message": "никогда" },
     "html_options_suspend_seconds": { "message": "секунд" },
-    "html_options_suspend_minute": { "message": "минуту" },
-    "html_options_suspend_minutes": { "message": "минуты" },
-    "html_options_suspend_hour": { "message": "час" },
+    "html_options_suspend_minute": { "message": "минуты" },
+    "html_options_suspend_minutes": { "message": "минут" },
+    "html_options_suspend_hour": { "message": "часа" },
     "html_options_suspend_hours": { "message": "часов" },
-    "html_options_suspend_day": { "message": "день" },
+    "html_options_suspend_day": { "message": "дня" },
     "html_options_suspend_days": { "message": "дней" },
-    "html_options_suspend_week": { "message": "неделю" },
-    "html_options_suspend_weeks": { "message": "недели" },
-    "html_options_suspend_no_pinned": { "message": "Никогда не приостанавливать закрепленные" },
+    "html_options_suspend_week": { "message": "недели" },
+    "html_options_suspend_weeks": { "message": "недель" },
+    "html_options_suspend_no_pinned": { "message": "Никогда не приостанавливать закрепленные вкладки" },
     "html_options_suspend_no_forms": { "message": "Никогда не приостанавливать вкладки с несохраненными полями ввода" },
     "html_options_suspend_no_audio": { "message": "Никогда не приостанавливать вкладки с проигрывающимся аудио" },
     "html_options_suspend_only_connected": { "message": "Никогда не приостанавливать вкладки в режиме оффлайн" },
     "html_options_suspend_only_on_battery": { "message": "Никогда не приостанавливать вкладки, когда подключен источник питания" },
-    "html_options_suspend_no_active_tabs": { "message": "Никогда не приостанавливать вкладки активную вкладку окна" },
-    "html_options_suspend_automatically_unsuspend": { "message": "Автоматически возобновлять вкладку при открытии" },
-    "html_options_suspend_discard_after_suspend": { "message": "Применить встроенный в Chrome механизм сохранения памяти при приостановке" },
-    "html_options_suspend_discard_after_suspend_tooltip_line1a": { "message": "Уменьшает использование памяти приостановленной вкладки еще больше, используя" },
-    "html_options_suspend_discard_after_suspend_tooltip_line1b": { "message": "встроенную в Chrome функциональность 'отбрасывание вкладки'. Это может привести к 500%" },
-    "html_options_suspend_discard_after_suspend_tooltip_line1c": { "message": "уменьшению использования памяти при работе с большим количеством вкладок." },
-    "html_options_suspend_discard_after_suspend_tooltip_line2a": { "message": "Однако, это может также привести к небольшой задержке отображения при выборе" },
+    "html_options_suspend_no_active_tabs": { "message": "Никогда не приостанавливать активную вкладку в каждом окне" },
+    "html_options_suspend_automatically_unsuspend": { "message": "Автоматически возобновлять вкладку при выделении" },
```

Comment:
..."при переходе на нее"?

Reply:
in my translation
Автоматически возобновлять приостановленную вкладку при переключении
why do i see above
Автоматически возобновлять вкладку при выделении
?

Reply:
Hmm I'm not sure, maybe coz I used ur initial commit to comment on, but anyway I recommend the use of "при переходе на нее" instead of "при выделении", more accurate in my view

Reply:
last version when opening a commit its "correction#4-1".
I do not like both word (переключение/выделение), yours is better, fix.

Reply:
![image](https://user-images.githubusercontent.com/23438485/52138612-b5c6d980-265e-11e9-86b7-da39b73a8c74.png)
I use this button, as far as I see all my changes there

Reply:
Thanks for explanation 😊